test/swap.js

```diff
@@ -44,8 +41,4 @@ describe('swap', function() {
   it('swaps characters in string from one index to another', function() {
     eq(R.swap(0, 2, 'foo'), 'oof');
   });
-
-  it('swaps object indexes in map from one index to another', function() {
-    eq(R.swap(o, o2, map), new Map([[o, 2], [o2, 1]]));
-  });
```

Comment:
Given the current `swap` implementation there is no way this could ever work. Unless I'm missing something.

Reply:
That's odd.  No problem removing it.